docs/INSTALL.md

```diff
@@ -345,4 +347,6 @@ Bitcoin chain. `lnd` also supports Litecoin testnet4 (but not both BTC and LTC
 at the same time), so when working with Litecoin be sure to set to parameters
 for Litecoin accordingly. For node configuration, the sections are called
 `[Btcd]`, `[Bitcoind]`, `[Neutrino]`, `[Ltcd]`, and `[Litecoind]` depending on
-which chain and node type you're using.
+which chain and node type you're using.  
+Make sure to place options in the proper section.  
```

Comment:
What I think would be nice would be another paragraph "getting started" or "first steps" that explains how to activate autopilot and/or connect to a node to be able to transfer in both directions. This is something for another PR, though.